src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.storage.StorageImpl;
+import core.basesyntax.strategy.handler.BalanceOperationHandler;
+import core.basesyntax.strategy.handler.OperationHandler;
+import core.basesyntax.strategy.handler.PurchaseOperationHandler;
+import core.basesyntax.strategy.handler.ReturnOperationHandler;
+import core.basesyntax.strategy.handler.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private static final FruitTransaction B_APPLE_100 =
+            new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", 100);
+    private static final FruitTransaction S_APPLE_50 =
+            new FruitTransaction(FruitTransaction.Operation.SUPPLY, "apple", 50);
+    private static final FruitTransaction R_APPLE_5 =
+            new FruitTransaction(FruitTransaction.Operation.RETURN, "apple", 5);
+    private static final FruitTransaction P_APPLE_30 =
+            new FruitTransaction(FruitTransaction.Operation.PURCHASE, "apple", 30);
+    private static final FruitTransaction B_ORANGE_40 =
+            new FruitTransaction(FruitTransaction.Operation.BALANCE, "orange", 40);
+
+    private OperationStrategy strategy;
+    private Storage storage;
+
+    @Before
+    public void setUp() {
+        storage = new StorageImpl();
```

Comment:
do you clear storage after tests? they should not depend on one another changes to storage